packages/types/src/interfaces.ts

```diff
@@ -2,6 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import {Extrinsic, EventRecord, SignedBlock} from '@polkadot/types/interfaces';
+import {BlockInfo, EventsByType} from '@terra-money/terra.js';
```

Comment:
if you introduce terra sdk to types package, you will need to add dependency in `types/package.json`.
Though this is not the preferable way, since no projects will rely on both polkadotjs and terra.js together.
I suggest a seperate `@subql/types-terra` for terra subquery project only.